libs/barista-components/drawer-table/src/drawer-table.scss

```diff
@@ -0,0 +1,18 @@
+$turquoise-600: #00a1b2;
+
+dt-drawer.dt-drawer-table-open:not(.dt-drawer-over) {
+  flex-grow: 100;
+}
+
+dt-drawer.dt-drawer-table-open.dt-drawer-over {
+  width: 100%;
+  border: none;
+}
+
+:host ::ng-deep .dt-drawer-row-def {
+  cursor: pointer;
+}
+
+:host ::ng-deep .dt-drawer-row-def-selected {
+  border: 1px solid $turquoise-600 !important;
```

Comment:
I do hope that we can do this without an important. `!important` is impossible to override. I would rather opt for a specificity increase with a selector like this: 
```scss
:host ::ng-deep .dt-drawer-row-def-selected.dt-drawer-row-def-selected  {
 border: 1px solid $turquoise-600;
}
```
This increases the rules specificity from 0020 to 0030 (important would increase it to 1020).

Depending on what you actually want to change about the selected row, it might be sufficient to change the `border-color`, instead of redefining the whole border. If we ever change the border styling of the selected row, the drawer-table would not benefit from that change and it would lead to increasing inconsistency.